go/libraries/doltcore/sqle/sqlutil/convert.go

```diff
@@ -97,6 +97,11 @@ func FromDoltSchema(tableName string, sch schema.Schema) (sql.Schema, error) {
 	var i int
 	_ = sch.GetAllCols().Iter(func(tag uint64, col schema.Column) (stop bool, err error) {
 		sqlType := col.TypeInfo.ToSqlType()
+		var extra string
+		if col.AutoIncrement {
+			extra = "auto_increment"
+		}
```

Comment:
@VinaiRachakonda this is was the engintest failure

Reply:
Where can I find the log for the failure?

Reply:
it's fixed now, but it came up in the go tests